en/README.md

```diff
@@ -0,0 +1,1138 @@
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### putTriggers(method)
+
+Send all triggers information to Hatohol server to use "ALL" option when complate conect to Hatohol server or reseive [fetchTriggers](#user-content-fetchtriggersmethod) procedure from Hatohol server.
+
+ In case of using "UPDATE" option, send diffarence to use lastInfo that got using [getLastInfo](#user-content-getlastinfomethod) or saved in internal to Hatohol server.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|triggers    |object array|M|-|This is object array that stores triggers information. Look at the following table for details.|
+|updateType|string    |M|-|Select sending option that match the situation from [[List](#user-content-updatetype)].|
+|lastInfo    |String32767|O|-|Insert to here trigger information of last. This information is result of [getLastInfo](#user-content-getlastinfomethod).|
+|fetchId|String255|O|-|This is ID that indicates whether response for request from Hatohol server. Insert fetchId value to this object, in case of only receive fetchTriggers request.
+
+***triggers object***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|triggerId     |String255  |M|-|Id of the trigger. When send trigger of HAP itself, must insert "_SELF_" to trigger Id and host Id.|
+|status        |string     |M|-|Status of trigger [[List](#user-content-triggerstatus)].|
+|severity      |string     |M|-|Severity of trigger [[List](#user-content-triggerseverity)].|
+|lastChangeTime|TimeStamp  |M|-|Time of last updated trigger.|
+|hostId        |String255  |M|-|Id of host that is trigger of host belongs.|
+|hostName      |String255  |M|-|Host name of host that is trigger of host belongs.|
+|brief         |String255  |M|-|Overview of trigger.|
+|extendedInfo  |String32767|M|-|The extra information other than the followings. It uses to show data on WebUI mainly.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "triggers": [
+      {
+        "extendedInfo": "sample extended info",
+        "brief": "example brief",
+        "hostName": "exampleName",
+        "hostId": "1",
+        "lastChangeTime": "20150323175800",
+        "severity": "INFO",
+        "status": "OK",
+        "triggerId": "1"
+      }
+    ],
+    "fetchId": "1",
+    "lastInfo": "201504061606",
+    "updateType": "UPDATED"
+  },
+  "method": "putTriggers",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Receive result as result object value whether sent data has been updated. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### putEvents(method)
+
+putEvents procedures can send till 1000 events. If want to send over than 1000, send events to separate more than once.
```

Comment:
> If want to send over than 1000, send events to separate more than once.

If users want to send over than 1000 event with putEvents procedure, users should be separate request which contains less than 1000 events and send them.